frontend/pages/artist/_itemId/index.vue

```diff
@@ -88,6 +97,114 @@
               </v-row>
             </v-tab-item>
             <v-tab-item :key="1">
+              <v-row no-gutters>
```

Comment:
Can you move the tab's content to a new component? That way we have less duplication. A fitting place is ``components/Layout/Pages/Artist/ArtistTab.vue`` in my opinion

Reply:
I've moved it over, though I didn't use the exact path suggested.